gulp/tasks/browserSync.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var util = require( 'gulp-util' );
+var browserSync = require( 'browser-sync' );
+
+gulp.task( 'browserSync', function() {
+  var port = util.env.port || '4000';
+  browserSync.init( {
+    proxy: 'localhost:' + port
+  } );
+} );
```

Comment:
I think we make use of some of the watch tasks, I'd be all for taking browsersync out here.